src/main/java/edu/ucsb/ucsbcslas/entities/OnlineOfficeHours.java

```diff
@@ -21,14 +21,19 @@
   @JoinColumn(name = "tutor_assignment_id")
   private TutorAssignment tutorAssignment;
   @Column(nullable = false)
+  @CsvBindByPosition(position = 0)
```

Comment:
Is this being used, as you directly parse the CSV file in your controller? If this is not needed, please remove all CsvBindByPosition annotations.